src/cats/cats.service.ts

```diff
@@ -0,0 +1,8 @@
+import { Injectable } from "@nestjs/common";
+
+@Injectable()
+export class CatService {
+    findAll(): string {
+        return 'This is cat world!.';
```

Comment:
- create an interface folder and create interface for cat
- don't just return plain string
- create a list of demo data of cats having type as defined in interface
- return list of above demo data